src/packages/end-to-end/package.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "@exogee/graphweaver-end-to-end",
+	"version": "0.0.1",
+	"description": "Graphweaver Test Suite",
+	"license": "MIT",
```

Comment:
The license at the root is "license": "UNLICENSED",
What's the license of GW? and does it need to be consistent within packages?

Reply:
everything should be MIT. I have updated the one in root.